libgraph/src/PropertyGraph.cpp

```diff
@@ -964,6 +968,29 @@ katana::PropertyGraph::DeleteEdgeIndex(const std::string& column_name) {
   return KATANA_ERROR(katana::ErrorCode::NotFound, "edge index not found");
 }
 
+// todo(Chak-Pong) expose it to python side
```

Comment:
Which python side? You seemed to be talking about `katana.remote` on slack.

Reply:
yes, the python side `katana.remote` used here:

https://github.com/KatanaGraph/katana-enterprise/blob/master/python/test/remote_graph_test.py